interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,155 @@
+                    <children>
+                      <leafNode name="axfr">
+                        <properties>
+                          <help>Source IP address</help>
+                          <valueHelp>
+                            <format>ipv4</format>
+                            <description>IPv4 address</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="ipv4-address"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="url">
+                        <properties>
+                          <help>Source URL</help>
+                          <valueHelp>
+                            <format>url</format>
+                            <description>Set URL location</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="url" argument="--scheme http --scheme https"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                    </children>
+                  </node>
+                  <node name="options">
+                    <properties>
+                      <help>Zone to cache options</help>
+                    </properties>
+                    <children>
+                      <leafNode name="timeout">
+                        <properties>
+                          <help>The maximum time a retrieval using the AXFR or URL source may take</help>
+                          <valueHelp>
+                            <format>u32:1-3600</format>
+                            <description>Request timeout in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-3600"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>20</defaultValue>
+                      </leafNode>
+                      <leafNode name="refresh">
+                        <properties>
+                          <help>The interval to wait between retrievals. A value of zero means the retrieval is done once at startup and on Lua configuration reload</help>
+                          <valueHelp>
+                            <format>u32:0-2147483647</format>
+                            <description>Refresh period in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 0-2147483647"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>86400</defaultValue>
+                      </leafNode>
+                      <leafNode name="retry-on-error">
```

Comment:
This option name sounds as if it's a boolean, but it's not. I think it should be `retry-interval`.

Reply:
done